model/time.go

```diff
@@ -202,13 +203,23 @@ func ParseDuration(durationStr string) (Duration, error) {
 
 	// Parse the match at pos `pos` in the regex and use `mult` to turn that
 	// into ms, then add that value to the total parsed duration.
+	var overflowErr error
 	m := func(pos int, mult time.Duration) {
 		if matches[pos] == "" {
 			return
 		}
 		n, _ := strconv.Atoi(matches[pos])
+
+		// Check if range overflow Time.duration ( > ~290years)
+		if n > int((1<<63-1)/mult/time.Millisecond) {
+			overflowErr = errors.New("Time range overflow: Query range exceeds 290 years limit)")
```

Comment:
```suggestion
			overflowErr = errors.New("Time range overflow: Query range exceeds 290 years limit)")
```

- The error messages should not be capitalized.
- The durations are not only used in queries.

I would like to have a simpler error message like `duration out of range` of `duration overflow`.